<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signalDisplay</class>
 <widget class="QWidget" name="signalDisplay">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="3">
    <widget class="QCustomPlot" name="waveformPlot" native="true"/>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="pauseButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="signalsAPP_resources.qrc">
       <normaloff>:/icon_images/pauseButton.png</normaloff>:/icon_images/pauseButton.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="settingsButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="signalsAPP_resources.qrc">
       <normaloff>:/icon_images/settingsIcon.jpg</normaloff>:/icon_images/settingsIcon.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="startButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="signalsAPP_resources.qrc">
       <normaloff>:/icon_images/playButton.png</normaloff>:/icon_images/playButton.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="modeBox">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-style:italic;&quot;&gt;You can select between &lt;/span&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; font-style:italic;&quot;&gt;3 Display Modes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="currentText">
      <string>Select Mode</string>
     </property>
     <item>
      <property name="text">
       <string>Select Mode</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Record</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Generate</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Load</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="outputDevice">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>0</height>
      </size>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Output: Not Set</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="inputDevice">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>0</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Input: Not Set</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="signalsAPP_resources.qrc"/>
 </resources>
 <connections/>
</ui>
